app/models/user.py

```diff
@@ -352,3 +355,19 @@ def __setattr__(self, name, value):
 @event.listens_for(User, 'init')
 def receive_init(target, args, kwargs):
     target.signup_at = datetime.now(pytz.utc)
+
+
+@event.listens_for(User, 'after_update')
```

Comment:
@poush I still find it a bad idea to do this task in model level rather than an API level. Model is supposed to do transactions and getter and setter tasks mostly. the event listener you are using is mainly used when you need to rollback a commit based on certain factors in the model change which is again a DB operation. So I find it a little bad idea. Had it been some logging function we were implementing I would have said kudos to your implementation. I will wait for @fossasia/open-event-server-devs  to decide.